src/chrome/content/rules/Statistics.gov.uk.xml

```diff
@@ -29,8 +29,14 @@
 		- www2.secure-transfer.statistics.gov.uk
 -->
 <ruleset name="Statistics.gov.uk (partial)">
+	<target host="www.statistics.gov.uk" />
+		<test url="http://www.statistics.gov.uk/about/national_statistics" />
 	<target host="neighbourhood.statistics.gov.uk" />
 	<target host="www.neighbourhood.statistics.gov.uk" />
+	
+	<rule from="^http://www\.statistics\.gov\.uk/.*"
+		  	to="https://www.gov.uk/government/statistics/announcements" />
```

Comment:
That behavior seems wrong: `http://www.statistics.gov.uk/invalid`

Reply:
Do you mean that I should also include the exclusion in the old ruleset?

Reply:
Yes, or you change the rule to only match specific urls.
And please remove is the `.*` in the expression.